package.json

```diff
@@ -1,8 +1,24 @@
 {
   "name": "style-loader",
   "version": "0.16.1",
-  "author": "Tobias Koppers @sokra",
   "description": "style loader module for webpack",
+  "engines": {
+    "node": ">= 0.12.0"
```

Comment:
Yep but this PR is minor and current `style-loader` is node =< 4 aswell, I bump it with the `webpack-defaults` upgrade 😛